frontend/src/modules/report/ReportContainer.jsx

```diff
@@ -180,17 +166,16 @@ class ReportContainer extends Component {
                 product={product}
                 version={version}
                 build={build}
-                selectedFeatureId={this.state.selectedFeature ? this.state.selectedFeature._id : null}
+                selectedFeatureId={selectedFeature?.id}
```

Comment:
The previous line was _id rather than id - is this correct?

Reply:
Yeah this is a mistake, will fix